### PR TITLE
add react project

### PR DESCRIPTION
**Pull request by ahmed-kino**
is there any comments you need to add for this pull request ?

### CODE REVIEW

**Comment by TheoLeanse** [comment]
@Ehmed-kino doesn't look like this will be using node

edit: but you're right, we can keep the react and the node apps separate

**Review by Md-Almasri — CHANGES_REQUESTED** [review]
delete unnecessary files

**Review by ahmed-kino — COMMENTED** [review]
@Md-Almasri  I deleted extra things from react folder and added really basic logo as well

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tag above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Md-Almasri) [inline public/index.html:16]: @Ehmed-kino could you please change the title?

**Review by Md-Almasri — APPROVED** [review]
Nice, could you merge your changes please?

**Pull request merged**